exercises/basics/strings.ts

```diff
@@ -52,9 +52,9 @@ export function regularExpressionsA() {
 export function stringMethods() {
   const dataA = 'Hello world';
   const dataB = 'table tennis';
-  const dataC = '\nmore spin\n ';
+  const dataC = ' \nmore spin\n ';
 
-  const indexA = -1;
+  const indexA = dataA.length - 1;
```

Comment:
it's ok to do `dataA.length` but it can lead to bugs down the road... better to just be explicit and put the actual numeric value of `10` @vuhungthang